api/kubernetes/deploy_via_helm-dev.sh

```diff
@@ -29,26 +29,171 @@ if [[ $# -lt 3 ]]; then
 fi
 
 export NAMESPACE="$1"
+export ARGO_POSTGRESQL_PASSWORD="$2"
 export API_VERSION="$3"
 
+export ARGO_RELEASE_NAME=argo
+export MINIO_RELEASE_NAME=minio
+export POSTGRES_RELEASE_NAME=postgres
+export ARGO_VERSION="0.16.6"
+export MINIO_VERSION="8.0.10"
+export POSTGRESQL_VERSION="11.6.12"
+
+export ARGO_DATABASE=argo_workflows
+export ARGO_POSTGRESQL_USER=argo
+export ARGO_BUCKET_NAME=argo-workflows
+
 HELM_CHARTS_BASE_PATH=$(realpath "$(dirname "$0")")
 
+WORKING_DIR=$(mktemp -d -t cosmotech-api-helm-XXXXXXXXXX)
+echo "[info] Working directory: ${WORKING_DIR}"
+pushd "${WORKING_DIR}"
+
 # Create namespace if it does not exist
 kubectl create namespace "${NAMESPACE:-phoenix}" --dry-run=client -o yaml | kubectl apply -f -
 
+# Minio
+cat <<EOF > values-minio.yaml
+fullnameOverride: argo-${MINIO_RELEASE_NAME}
+defaultBucket:
+  enabled: true
+  name: ${ARGO_BUCKET_NAME}
+persistence:
+  enabled: true
+resources:
+  requests:
+    memory: "2Gi"
+service:
+  type: ClusterIP
+DeploymentUpdate:
+  # As Minio uses a ReadWriteOnce PVC by default, using a RollingUpdate strategy will not
+  # work if the new pod is scheduled on a different node.
+  # It is possible to force the replicas scheduling on a same node, but this requires that node
+  # to have a minimum of 4Gi additional memory available, which is the default memory request
+  # set by Argo.
+  type: Recreate
+networkPolicy:
+  # Enabling networking policy returns the following error: unable to recognize "": no matches for kind "NetworkPolicy" in version "networking.k8s.io/v1beta1"
+  # => will use the 'networking/traffic-allowed: "yes"' label instead to allow traffic.
+  enabled: false
+  allowExternal: true
+podLabels:
+  networking/traffic-allowed: "yes"
+
+EOF
+
+helm repo add minio https://helm.min.io/
+helm upgrade --install ${MINIO_RELEASE_NAME} minio/minio --namespace ${NAMESPACE} --version ${MINIO_VERSION} --values values-minio.yaml
+
+# Postgres
+cat <<EOF > values-postgresql.yaml
+auth:
+  username: "${ARGO_POSTGRESQL_USER}"
+  password: "${ARGO_POSTGRESQL_PASSWORD}"
+  database: ${ARGO_DATABASE}
+primary:
+  podLabels:
+    "networking/traffic-allowed": "yes"
+
+EOF
+
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm upgrade --install -n ${NAMESPACE} ${POSTGRES_RELEASE_NAME} bitnami/postgresql --version ${POSTGRESQL_VERSION} --values values-postgresql.yaml
+
+export ARGO_POSTGRESQL_SECRET=argo-postgres-config
```

Comment:
Could be interesting to rename `ARGO_POSTGRESQL_SECRET` to `ARGO_POSTGRESQL_SECRET_NAME` or somtehing like this. It would be clearer at read.